lib/active_remote/rpc.rb

```diff
@@ -53,8 +53,11 @@ def rpc_adapter
     end
 
     def assign_attributes_from_rpc(response)
-      new_attributes = self.class.build_from_rpc(response.to_hash)
-      @attributes.update(new_attributes)
+      unless response.respond_to?(:errors) && response.errors.count > 0
```

Comment:
Would it be better to exit the method early vs wrapping it in an `unless`?

```ruby
return if response.respond_to?(:errors) && response.errors.count > 0
```

Reply:
But we would skip the `add_errors` later in the function, we may not want that to happen.

Reply:
I still want to make sure that the errors get appended a couple lines below.

Reply:
Yeah, sorry. I didn't see line 61.